### DISCUSSION TITLE
🆕 New in GitBook: Page icons, docs site improvements, hidden pages and more

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#675] [Announcements]
Hi everyone! We're back again this month with a *ton* of updates for you and your published docs. 

![image](https://github.com/user-attachments/assets/ea402be6-b89d-43e4-8d4b-f84519a0fbeb)

Let's see what's new: 

**📫 Published Docs Updates 📫** 
Over the last few weeks, we've been rolling out new features to make it easier for you and your team to manage and create published documentation in GitBook. With a completely re-vamped UI and workflow to create sites, extra customization options, and a better view into the settings for your sites, it's never been easier to publish beautiful documentation with GitBook. Head to our [blog post](https://www.gitbook.com/blog/new-introducing-docs-sites) to read everything that's new.

**🕵️ Hidden Pages 🕵️** 
A highly requested feature is now available for everyone! You're able to hide pages from the table of contents, while still being able to link to them across other pages and sites. Head to a page or group's options to hide it from view.

**❇️ Page Icons ❇️** 
Another highly requested feature is now available in-app too: Page Icons! Choose from a set of 3,600 icons to add to your doc's table of contents and the top of pages in GitBook. 

We're also hosting a webinar later today to dive into all of the things that changed! Register to tune in live or get the recording [here](https://www.gitbook.com/events/gitbook-for-public-docs).

Hop on over to our [changelog](https://changelog.gitbook.com/) to see all the latest changes.

**Comment by xavier-chiliz** [comment]
Solid upgrade!
I especially appreciate the new "Hide page", which is the next best thing to having a proper "Draft" mode (#423).
Thank you the team!

**Comment by augustoortolantr** [comment]
Great update, however, when I use search, the pages that are hidden appear in the search.
Is there any fix for this planned?

**Comment by eric-maniloff** [comment]
the favicons of our pages don't show anymore, instead it shows "E" (for english I think?) . any way to get our brand's icons back?? 
![image](https://github.com/user-attachments/assets/68b1edb8-9415-4763-824b-858cc1fd04ff)

**Comment by strickvl** [comment]
Where are the page icons to be found? i.e. a canonical list?

**Comment by JFQueralt** [comment]
Is there a way to get the icon set somewhere?
For visual consistency we would like to build some icons that we'll use in our documentation pages and in a few places more.
At the moment even downloading them one at a time is not doable.

**Reply by JFQueralt** [reply]
OK, I see they are obtained via https://fontawesome.com/
